Mesh/Mesh.cpp

```diff
@@ -979,10 +976,14 @@ void Mesh::ConstructSubDomain_by_Nodes(const MeshPartConfig mpc)
    vector<Node*> sbd_nodes_hq;     // nodes of a partition for quadratic element
    std::vector<std::vector<std::set<ConnEdge> > > vec_neighbors(num_parts); //NW
 
+   // Should be removed for openmp 
+   for (std::size_t i=0; i<elem_vector.size(); i++)

```

Comment:
You could use a [range-based for loop](http://en.cppreference.com/w/cpp/language/range-for).